projects/demo/src/modules/components/input-date-time/examples/4/value-transformer.ts

```diff
@@ -0,0 +1,24 @@
+import {AbstractTuiControlValueTransformer, TuiDay, TuiTime} from '@taiga-ui/cdk';
+
+export class ExampleDateTimeTransformer extends AbstractTuiControlValueTransformer<
+    [TuiDay | null, TuiTime | null],
+    string
+> {
+    private readonly separator = ', ';
+
+    fromControlValue(controlValue: string): [TuiDay | null, TuiTime | null] {
```

Comment:
Please use ternary operators in both functions

Reply:
done